guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Hammer-cli.adoc

```diff
@@ -0,0 +1,31 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Hammer_CLI_{context}"]
+= Using IdM credentials to log in to the {Project} Hammer CLI
+
+This section describes how to log in to your {Project} Hammer command-line interface (CLI) with your {FreeIPA} (IdM) login and password.
```

Comment:
```suggestion
This section describes how to log in to your {Project} Hammer (CLI) with your {FreeIPA} (IdM) login and password.
```

Reply:
Should the brackets remain around 'CLI'?

Reply:
No, it's "Hammer CLI".

Reply:
OK thanks for confirming